www/pages/plan-trip/plan-trip-controller.js

```diff
@@ -443,6 +457,12 @@ angular.module('pvta.controllers').controller('PlanTripController', function ($s
     $scope.params.time.type = selectedOption.type;
   };
 
+  $scope.goToStop = function(loc) {
+    NearestStop.get({latitude: loc.lat(), longitude: loc.lng()}).$promise.then(function(stop) {
```

Comment:
Why force `NearestStop` to return a promise and then immediately resolve it?  I think you could just write `NearestStop.get(...).then(...);